app/_hub/kong-inc/rate-limiting-advanced/index.md

```diff
@@ -116,14 +123,6 @@ params:
 
         In Konnect Cloud, the default strategy is `redis`.
 
-        {:.important}
```

Comment:
Add missing strategy to the `config.strategy` configuration property:

"local: Counters are stored locally in-memory on the node (same effect as setting `sync_rate` to `-1`."